### ISSUE TITLE
Cannot resolvev 

### ISSUE THREAD

**Issue by SteveALee** [date-fns/date-fns#666]
`Module not found: Can't resolve 'date-fns/esm/locale' in ...'`

I'm following the i18n example in the docs

```
import { es, ru } from 'date-fns/esm/locale'
```

In a Create React App generated project (ie webpack)

The esm folder does not exist in node_modules and I can only find it as a `module` value in package.json

How should we import locales?

**Comment by SteveALee** [comment]
`import en from 'date-fns/locale/en'` does seem to work

However, I expect the local to specify a default format (like Windows regional settings) but it doesn't so is of less use to me.

**Comment by tom4dev** [comment]
I confirm, there is an error in examples available at https://date-fns.org/:

**Replace**

```import { fr } from 'date-fns/esm/locale'```

**By**
```import fr from 'date-fns/locale/fr';```

And it will works like a charm 👌

**Comment by mesqueeb** [comment]
I also found an error at: https://date-fns.org/v2.0.0-alpha.7/docs/I18n

At this line:
`import {en, eo, ru} from 'date-fns/esm/locale'`

Because when I looked in the [soure](https://github.com/date-fns/date-fns/blob/master/src/esm/locale/index.js) there's no actual `en`, and it looks like you have to write `enUS` or something instead.

I'm just confused why not all languages are listed here like `ja` and others.

**Comment by packetstracer** [comment]
this doesn't work for me

`import {eo} from 'date-fns/esm/locale'`

cannot find a folder named 'esm' or something similar regarding to ES modules.

The only thing that worked is using several import statements as @tom4dev suggested, but this seems a little verbose to me though.

`import en from 'date-fns/locale/en'
import ru from 'date-fns/locale/ru'`

**Comment by FerhatACAR** [comment]
> I also found an error at: https://date-fns.org/v2.0.0-alpha.7/docs/I18n
> 
> At this line:
> `import {en, eo, ru} from 'date-fns/esm/locale'`
> 
> Because when I looked in the [soure](https://github.com/date-fns/date-fns/blob/master/src/esm/locale/index.js) there's no actual `en`, and it looks like you have to write `enUS` or something instead.
> 
> I'm just confused why not all languages are listed here like `ja` and others.

Yes the problem is clear.It cant find en because you can see there is no folder named en.You have to use enUS, enCA or enGB

`import enUS from 'date-fns/locale';`

import your language like this.
